src/HealthChecks.Prometheus.Metrics/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -1,17 +1,20 @@
-﻿using HealthChecks.Prometheus.Metrics;
+﻿using System;
+using HealthChecks.Prometheus.Metrics;
 using Microsoft.AspNetCore.Diagnostics.HealthChecks;
 using Microsoft.AspNetCore.Http;
 
 namespace Microsoft.AspNetCore.Builder
 {
     public static class PrometheusHealthCheckMiddleware 
     {
-        public static IApplicationBuilder UseHealthChecksPrometheusExporter(this IApplicationBuilder applicationBuilder, PathString endpoint) 
+        public static IApplicationBuilder UseHealthChecksPrometheusExporter(this IApplicationBuilder applicationBuilder, PathString endpoint, Action<HealthCheckOptions> configure = null)

```

Comment:
This change is still binary breaking.

Reply:
Ok.  Would it be better if I added the `Action<HealthCheckOptions>` parameter as an overloaded method instead?

Reply:
This is usually done when you want to guarantee 100% backward compatibility.

Reply:
Moved options to an overloaded method.